doc/tech/glossary.md

```diff
@@ -1,37 +1,37 @@
 [<< Back](../../ref_model)
-# Glossary - Terminology
-<p align="right"><img src="../figures/bogo_lsf.png" alt="scope" title="Scope" width="35%"/></p>
+# Glossary
+<p align="right"><img src="./figures/bogo_lsf.png" alt="scope" title="baldy" width="35%"/></p>
 
 ## Table of Contents
-* [11.1 Terminology](#11.1)
-  * [11.1.1 Software layers terminology](#11.1.1)
-  * [11.1.2 Hardware layers terminology](#11.1.2)
-  * [11.1.3 Operational and administrative terminology](#11.1.3)
-  * [11.1.4 Container Related Terminology](#11.1.4)
-  * [11.1.5 Other terminology](#11.1.5)
+* [1. Terminology](#1.0)
+  * [1.1 Software layers terminology](#1.1)
+  * [1.2 Hardware layers terminology](#1.2)
+  * [1.3 Operational and administrative terminology](#1.3)
+  * [1.4 Container Related Terminology](#1.4)
+  * [1.5 Other terminology](#1.5)
 
-<a name="11.1"></a>
-## 11.1 Terminology
+<a name="1.0"></a>
+# 1 Terminology
 
 To help guide the reader, this glossary provides an introduction to the terminology used within this document. These definitions are, with a few exceptions, based on the ETSI GS NFV 003 V1.4.1 (2018-08) definitions.  In a few cases, they have been modified to avoid deployment technology dependencies only when it seems necessary to avoid confusion.
 
-<a name="11.1.1"></a>
-### 11.1.1 Software Layer Terminology
+<a name="1.1"></a>
+## 1.1 Software Layer Terminology
 
 - **Network Function Virtualisation (NFV)**: The concept of separating network functions from the hardware they run on by using a virtual hardware abstraction layer.
-- **Network Function Virtualisation Infrastructure (NFVI)**: The totality of all hardware and software components used to build the environment in which a set of VAs are deployed.
- >_*Note:*_ The NFV-Infrastructure can span across many locations, e.g. places where data centres or edge nodes are operated. The network providing connectivity between these locations is regarded to be part of the NFVI. NFVI and VNF are the top-level conceptual entities in the scope of Network Function Virtualisation. All other components are sub-entities of these two main entities.
+- **Network Function Virtualisation Infrastructure (NFVI)**: The totality of all hardware and software components used to build the environment in which a set of VAs are deployed (also referred to as cloud infrastructure).
```

Comment:
VA?

Reply:
Virtual Application.

Reply:
Ah, okay, and it is in the glossary already.